src/components/Editor/index.js

```diff
@@ -245,7 +245,7 @@ class Editor extends PureComponent<Props, State> {
     // Only update and jump around in real source texts. This will
     // keep the jump state around until the real source text is
     // loaded.
-    if (selectedSource && selectedSource.has("text")) {
+    if (selectedSource && selectedSource.get("loadedState") === "loaded") {
```

Comment:
lets use the `isLoaded` [helper](https://github.com/devtools-html/debugger.html/blob/master/src/utils/source.js#L245-L247)